.build-emscripten.sh

```diff
@@ -25,6 +25,9 @@ pushd h3c
 git pull origin master --tags
 git checkout "v$(cat ../H3_VERSION)"
 
+# Run cmake - necessary for building h3api.h
+cmake .
```

Comment:
This is a downstream effect of https://github.com/uber/h3/pull/173 - may be relevant to other bindings as well.